hw3/01-characters.js

```diff
@@ -0,0 +1,33 @@
+const character = document.getElementsByClassName("character");
+
+async function getCharacters() {
+  const url = "https://thronesapi.com/api/v2/Characters";
+  try {
+    let response = await fetch(url);
+    return await response.json();
+  } catch (erorr) {
+    console.log(error);
+  }
+}
+
+async function renderCharacters() {
+  let characters = await getCharacters();
+  let html = "";
+
+  characters.forEach((character) => {
+    let htmlSegment = `<div class="character" onmouseover="this.style.background='var(--blue)';" onmouseout="this.style.background='none';">
```

Comment:
You can even use the .createElement and add bootstrap styling classes with the .classList.add() method to style the elements you want instead of writing html.